repos/spack_repo/builtin/packages/libmesh/package.py

```diff
@@ -274,7 +276,7 @@ def configure_args(self):
             options.append("--disable-metis")
 
         if self.spec.satisfies("+petsc") or self.spec.satisfies("+slepc"):
-            options.append("--enable-petsc=yes")
+            options.append("--enable-petsc-required")
```

Comment:
@capitalaslash, you should add a justification why this change is needed to the PR or the code.